### PR TITLE
Updating information on CIP1

### PR DESCRIPTION
**Pull request by davestaxcp**
A few spots of CIP1 pointed towards the now offline (old) Counterparty Forums. 

I changed the links to the new forum links that denote the correct area the CIP1 wanted to point new developers and users to.

This includes:
The Counterparty Forums (fixed link)
The CIP Discussions github
The Counterparty Developer Telegram

I also changed the last link which denoted sending Devon Weller an email when JPJA is now the CIP Editor.

### CODE REVIEW

File: cip-0001.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ The CIP editors assign CIP numbers and change their status. Please send all CIP-
...
 
-Each CIP must have a champion -- someone who writes the CIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The CIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is CIP-able. Posting to the [Counterparty forums](https://counterpartytalk.org/c/development) or discussing it in the [Counterparty Slack](http://slack.counterparty.io/) is the best way to go about this.
+Each CIP must have a champion -- someone who writes the CIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The CIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is CIP-able. Posting to the [Counterparty Forums](https://forums.counterparty.io/c/development/), posting it in the [Counterparty Github Discussions](https://github.com/CounterpartyXCP/cips/discussions) or discussing it in the [Counterparty Developer Telegram](https://t.me/Counterparty_Dev) is the best way to go about this.
```
Comment (jdogresorg) [inline cip-0001.md:32]: Perhaps remove the link to the forums and just leave the link to the github discussions?

File: cip-0001.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ The CIP editors assign CIP numbers and change their status. Please send all CIP-
...
+Once the champion has asked the Counterparty community as to whether an idea has any chance of acceptance, a draft CIP should be presented on a dedicated thread created by the author in the [Counterparty forums' development area](https://forums.counterparty.io/c/development/). This gives the author a chance to flesh out the draft CIP and to address initial concerns about the proposal. Discussion should be limited to this forum thread and as much as possible at this time, so as to provide a full record of any debate for all involved parties. Wherever possible, long open-ended discussions should be avoided, and CIP authors should use their discretion in order to best keep all debate focused, good-natured and material.
```
Comment (jdogresorg) [inline cip-0001.md:36]: Perhaps replace forums link with link to github discussions.

**Comment by jotapea** [comment]
See https://github.com/CounterpartyXCP/cips/discussions/97.

**Comment by davestaxcp** [comment]
Added a new commit to reflect @jdogresorg 's requests

@jotapea - None of these links in this commit use the wording "official" which you pointed out with #97  - which for the record I disagree with. It's important for the official links that developers and users use to go to the recommended discussion areas so that new users/devs know this is the officially recommended links from the Counterparty Docs and CIP's.... but left it without the word "official" for your liking here.

**Pull request closed without merging**